package/boot/uboot-sunxi/Makefile

```diff
@@ -9,9 +9,9 @@
 include $(TOPDIR)/rules.mk
 include $(INCLUDE_DIR)/kernel.mk
 
-PKG_VERSION:=2016.03
+PKG_VERSION:=2017.07
```

Comment:
2017.09 is available

Reply:
master now uses uboot 2017.07 in master, uboot 2017.09 needs some more testing before it could be merged and I wanted to patchset into the master.

Reply:
just mentioned about it, I started a build to test it